### PR TITLE
Add deploy script

### PR DESCRIPTION
**Pull request by iHiD**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        id: login_to_ecr
+        uses: aws-actions/amazon-ecr-login@v1
+
+      ###################
+      ###################
+      # Build the image #
+      ###################
+      ###################
+      - name: Build and Push Docker image
+        id: build_rails
```
Comment (ErikSchierboom) [inline .github/workflows/deploy.yml:52]: Does this step need an ID?

**Review by ErikSchierboom — APPROVED** [review]
This looks good! This is likely the thing you want me to script the PR's for, right? :)

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (iHiD) [reply .github/workflows/deploy.yml]: No. Good spot.

**Comment by iHiD** [comment]
> This is likely the thing you want me to script the PR's for, right? :)

Yes 🦜

**Pull request merged**